### PR TITLE
Fixed sidebar

### PR DESCRIPTION
**Pull request by TheBaljitSingh**
Homepage: Updated Homepage Title from EventManagement to EventSchedular
Sidebar: Updated the sidebar for Past Event
Event View: when user view past event added Banner for it. (have to take a fixed size banner from user then size issue will be done).
Removed Total Registration count from bottom of web pasted in sidebar

### CODE REVIEW

File: src/components/HomePagePastEvent.jsx
Comment (k1174) [inline src/components/HomePagePastEvent.jsx:1]: yeh to shi hai, fir sub me change karna padega. (rehnede dete hai)

image

File: src/components/Job.jsx [hunk 1]
```diff
@@ -54,11 +66,44 @@ const Job = ({ job }) => {
...
+
     return (
         <>
 
             <section className="bg-indigo-50">
-                <div className="container m-auto py-10 px-6">
+                <div className=" w-full h-[450px]">
+                    {/* this is for event banner image */}
+
+                    <img className="object-fill w-full max-h-[450px]" src={`${job.images[0]}`} alt="" />
```
Comment (k1174) [inline src/components/Job.jsx:99]: Handle if image is not uploaded.
Agar bola hai to dekhte hai, nhi to abhi ke liye rehde de.
(styling shi karna padega)

yeh ratio mujhe mila tha uski website se of img 
`img[Attributes Style] {
    width: 956px;
    aspect-ratio: auto 956 / 298;
    height: 298px;
}`

[Link of banner img](https://www.proelevate.in/_next/image?url=https%3A%2F%2Fcdn.proelevate.in%2FprofileImages%2F1725174568476_arnu6h2nw9.jpeg&w=3840&q=75)

File: src/components/Job.jsx [hunk 2]
```diff
@@ -74,13 +119,12 @@ const Job = ({ job }) => {
...
                                         className="fa-solid fa-location-dot text-lg text-orange-700 mr-2"
                                     ></i>
-                                    <p className="text-orange-700">{job.location}</p>
+
+
                                 </div>
                             </div>
 
-                            {job.images.length > 0 &&
-                                <Carousel items={job.images} />
```
Comment (k1174) [inline src/components/Job.jsx:82]: Images ko sir ne hi kaha tha ki uper lelo.

File: src/components/Job.jsx [hunk 3]
```diff
@@ -98,43 +142,183 @@ const Job = ({ job }) => {
...
+            </button>
+            {copied && <div className="absolute  bottom-full mb-2 left-1/2 transform -translate-x-1/2 bg-black text-white text-center text-sm rounded py-1 px-2">
+                {showTooltip === 'copy' && <>Copy</>}
+                {showTooltip === 'Copied!' && <>Copied!</>}
+            </div>
+    }
+        </div>
+                                                </div>
+                                                
+                                                <p className="font-medium text-sm">{job.organiserEmail}</p>
```
Comment (k1174) [inline src/components/Job.jsx:227]: overflow-wrap: anywhere;
Kar sakte hai.

image

File: src/components/Job.jsx (hunk 2 above)
Comment (TheBaljitSingh) [reply src/components/Job.jsx:82]: Updated sidebar ke saath image niche hi sahi dikh raha hai.

**Comment by TheBaljitSingh** [comment]
sir told me to that images should open in large size. so done

**Pull request merged**